app/types/index.ts

```diff
@@ -0,0 +1,32 @@
+export interface EmailInterface {
```

Comment:
I added a `types` directory to store some interfaces. These weren't available in the `generated/graphql.tsx`. I'll to research that more later.